src/app/services/public-comment.service.ts

```diff
@@ -0,0 +1,111 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
+import { PublicComment } from '../models/publiccomment';
+import { Comment } from '../models/comment';
+import { Document } from '../models/document';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/switchMap';
+
+import { Api } from './api';
+import { ValuedComponent } from '../models/vcs';
+
+@Injectable()
+export class PublicCommentService {
+  pcp: PublicComment;
```

Comment:
Question about what a "PublicComment" is.  Is it a single comment? Or a comment period?